### PR TITLE
Fix error on signon login, need create a cookie with sessionID

### PR DESCRIPTION
**Pull request by ricardojlrufino**

### CODE REVIEW

File: examples/signon.php [hunk 1]
```diff
@@ -30,6 +30,7 @@
     /* Update another field of server configuration */
     $_SESSION['PMA_single_signon_cfgupdate'] = array('verbose' => 'Signon test');
     $id = session_id();
+    setcookie($session_name, $id , 0, "/");
```
Comment (nijel) [inline examples/signon.php]: The `session_start` is supposed to do this...

Comment (ricardojlrufino) [reply examples/signon.php]: Strange, it only worked after I did this. 
After about 3 hours trying
``
Versão do cliente de banco de dados: libmysql - 5.5.44-MariaDB
Extensão do PHP: mysqli Documentação
Versão do PHP: 5.4.45
```

Comment (nijel) [reply examples/signon.php]: What is your setting for `session.use_cookies`? Does calling following prior to session_start fix your problem?

```php
@ini_set('session.use_cookies', 'true');
```

**Comment by nijel** [comment]
Closing as this should not be really needed. There might be other issue in the script, but this is certainly not the correct solution.

**Pull request closed without merging**